src/popper/runner_host.py

```diff
@@ -280,8 +280,6 @@ def run(self, step):
         if e != 0:
             log.fail(f"{output}")
 
-        print(output)
```

Comment:
@ivotron This is where I added a print statement to print the results after swallowing the outputs. I now understand the importance of having a test to check stdout is actually being captured by the logger.